### PR TITLE
Add Cargill copyright header to Splinter-docs

### PR DESCRIPTION
**Pull request by shannynalayna**
Also adds a newline to the end of a few .svg logo images.

Signed-off-by: Shannyn Telander <telander@bitwise.io>

### CODE REVIEW

**Comment by shannynalayna** [comment]
> Looks like the gameroom images are broken on this PR (checked master and that still looks fine)

Hm, I didn't notice this before, must have been looking at a cached version of the site. I'm going to have to figure out a different way to incorporate the copyright into the svg metadata. Interesting that this issue doesn't propagate to all svg images.

**Comment by vaporos** [comment]
The man page updates should go into the splinter repo first, and we should update them via the just copy-docs script in a separate PR.

File: docs/gameroom_walkthrough/act_1/scene_1.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+## Scene 1: Alice logs into Acme's Gameroom application
+
+Alice looks at the GAMEROOM APP LOGIN SCREEN in her browser.
```
Comment (agunde406) [inline docs/gameroom_walkthrough/act_1/scene_1.md:18]: Looks like you accidentally added this file back (it has been moved to each release and combined with the background part)

Comment (shannynalayna) [reply docs/gameroom_walkthrough/act_1/scene_1.md]: Updated

**Pull request merged**